configure.ac

```diff
@@ -267,29 +283,42 @@ AS_HELP_STRING([--with-fftw-dir=DIR],[Declare the root directory of fftw3, if it
     fi
 ])
 
-# Compile programs/samples/sample_functional_test.cpp 
-want_samples="no"
-AC_ARG_ENABLE(samples, AS_HELP_STRING([--enable-samples],[Compile with samples [default=no]]),[
-  if test "$enableval" = yes; then
-    want_samples="yes"
-  	AC_MSG_NOTICE([Compiling sample functional tests])	
-  fi
-  ]) 
-AM_CONDITIONAL([ENABLESAMPLES_AM], [test "$want_samples" = "yes"])
 
 # Use fine grained profiling
 
 want_profiling="no"  
 AC_ARG_ENABLE(profiling, AS_HELP_STRING([--enable-profiling],[Include fine-grained profiling [default=no]]),[
   if test "$enableval" = yes; then
-    AC_DEFINE([PROFILING], [], [Define the profiling flag])
+    AC_DEFINE([CISTEM_PROFILING], [], [Define the profiling flag])
   	want_profiling="true"
   	AC_MSG_NOTICE([Compiling with fine-grained profiling])	
   fi
   ]) 
 
 AM_CONDITIONAL([ENABLEPROFILING_AM], [test x$want_profiling = xtrue])
 
+AC_ARG_ENABLE(fp16-particlestacks, AS_HELP_STRING([--enable-fp16-particlestacks],[Save particle stacks in reduced precision. Use with caution  [default=no]]),[
```

Comment:
If the idea is that in the beginning these experimental features are only used by experts, maybe a command-line  flag might be a good option. It is easy to implement and they can easily be turned on and off by modifying the run profile.

Reply:
But that wouldn't work with the GUI, which is where this is used in this case. I agree for thinks like prepare_statck_matchtemplate, this would be a good candidate for a CL-arg

Reply:
Oh yes, the GUI is writing the particlestacks. I always wondered whether it would make sense at some point to move this to a cistem program (and have a "Generate RefinementPackage" job). That would make it easier to parallelize (which might help with the IO) and it would be less akward on clusters where the GUI runs on the headnode. 

Anyway, not a concern for this PR.